dgm-builder/src/main/java/com/cloudbees/groovy/cps/tool/Translator.java

```diff
@@ -294,7 +297,9 @@ private void translateMethod(final CompilationUnitTree cut, ExecutableElement e,
         }
         delegatingParams.forEach(p -> delegateCall.arg(p));
 
-        JVar $b = m.body().decl($Builder, "b", JExpr._new($Builder).arg(JExpr.invoke("loc").arg(methodName)));
+        JVar untrustedB = m.body().decl($Builder, "untrustedB", JExpr._new($Builder).arg(JExpr.invoke("loc").arg(methodName)));
+        JVar $b = m.body().decl($Builder, "b", untrustedB.invoke("contextualize").arg(codeModel.ref("com.cloudbees.groovy.cps.sandbox.Trusted").staticRef("INSTANCE")));
```

Comment:
It's basically avoiding needing to make alternate versions of every `Builder` method for "do this but automatically `Trusted`" by instead creating a copy of a normal `Builder` (which may or may not be `Trusted`/sandboxed) that *is* guaranteed to be `Trusted` and using that for everything but `Closure.call`.